Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-edit/user-edit.component.ts

```diff
@@ -66,43 +70,15 @@ export class UserEditComponent {
 
         }, err => {
```

Comment:
now I have added console.log(err.statusText) for the showing error. 
I think it not for the end user.